sql/src/main/java/io/crate/execution/support/OneRowActionListener.java

```diff
@@ -45,7 +45,14 @@ public OneRowActionListener(RowConsumer consumer, Function<? super Response, ? e
 
     @Override
     public void onResponse(Response response) {
-        consumer.accept(InMemoryBatchIterator.of(toRowFunction.apply(response), SENTINEL), null);
+        final Row row;
```

Comment:
Is this change related to the views?

Reply:
Yes, see https://github.com/crate/crate/pull/7047/files/2d9392826523b6ec74cc501b9457b2a36a93c43d#diff-798d69a5d85aa135cae9d6018cc0e4f1R64